src/k8s/api/v1/types.go

```diff
@@ -69,94 +98,72 @@ func (c ClusterStatus) HaClusterFormed() bool {
 
 // TICS -COV_GO_SUPPRESSED_ERROR
 // we are just formatting the output for the k8s status command, it is ok to ignore failures from result.WriteString()
-func (c ClusterStatus) datastoreToString() string {
-	result := strings.Builder{}
-
-	// Datastore
-	if c.Datastore.Type != "" {
-		result.WriteString(fmt.Sprintf("  type: %s\n", c.Datastore.Type))
-		// Datastore URL for external only
-		if c.Datastore.Type == "external" {
-			result.WriteString(fmt.Sprintln("  servers:"))
-			for _, serverURL := range c.Datastore.Servers {
-				result.WriteString(fmt.Sprintf("    - %s\n", serverURL))
-			}
-			return result.String()
-		}
-	}
-
-	// Datastore roles for dqlite
-	voters := make([]NodeStatus, 0, len(c.Members))
-	standBys := make([]NodeStatus, 0, len(c.Members))
-	spares := make([]NodeStatus, 0, len(c.Members))
-	for _, node := range c.Members {
-		switch node.DatastoreRole {
-		case DatastoreRoleVoter:
-			voters = append(voters, node)
-		case DatastoreRoleStandBy:
-			standBys = append(standBys, node)
-		case DatastoreRoleSpare:
-			spares = append(spares, node)
-		}
-	}
-	if len(voters) > 0 {
-		result.WriteString("  voter-nodes:\n")
-		for _, voter := range voters {
-			result.WriteString(fmt.Sprintf("    - %s\n", voter.Address))
-		}
-	} else {
-		result.WriteString("  voter-nodes: none\n")
-	}
-	if len(standBys) > 0 {
-		result.WriteString("  standby-nodes:\n")
-		for _, standBy := range standBys {
-			result.WriteString(fmt.Sprintf("    - %s\n", standBy.Address))
-		}
-	} else {
-		result.WriteString("  standby-nodes: none\n")
-	}
-	if len(spares) > 0 {
-		result.WriteString("  spare-nodes:\n")
-		for _, spare := range spares {
-			result.WriteString(fmt.Sprintf("    - %s\n", spare.Address))
-		}
-	} else {
-		result.WriteString("  spare-nodes: none\n")
-	}
-
-	return result.String()
-}
 
 // TODO: Print k8s version. However, multiple nodes can run different version, so we would need to query all nodes.
 func (c ClusterStatus) String() string {
 	result := strings.Builder{}
 
+	// longer than the longest key (eye-balled), make the output left aligned
+	maxLen := 25
+
 	// Status
 	if c.Ready {
-		result.WriteString("status: ready")
+		result.WriteString(fmt.Sprintf("%-*s %s", maxLen, "cluster status:", "ready"))
+	} else {
+		result.WriteString(fmt.Sprintf("%-*s %s", maxLen, "cluster status:", "not ready"))
+	}
```

Comment:
i think this would read better as `fmt.Sprintf("%25s %s", "cluster status:", "ready")` (it's ok to repeat 25 a few times in the code here)